### PR TITLE
crypt instead of wtw.aC5paV.qU

### PR DESCRIPTION
**Pull request by MaxiReglisse**
utilisation de la commande perl crypt pour générer le mot de passe chiffré de geoserver.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Is the `perl` command available in the base image ?

**Comment by MaxiReglisse** [comment]
yes, it works for me

**Comment by fvanderbiest** [comment]
LGTM.

@Vampouille are you OK ?

**Comment by Vampouille** [comment]
tested and working ! LGTM

**Comment by Vampouille** [comment]
Maybe you can remove this line : 

https://github.com/georchestra/georchestra/pull/1442/files#diff-15908953a1f1eaf343fa7a3b468a84d1R10

or use something like :
```bash
RUN useradd -ms /bin/bash --home /home/geoserver -p $(perl -e'print crypt("${USER_PASS:-geoserver}", "salt")') --uid 999 --gid 999 geoserver
```

**Comment by MaxiReglisse** [comment]
yes, it is better !

File: docker/ssh_data/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
 # add a user 'geoserver' with default password : 'geoserver'
 # that can be overriden by environement variable $USER_PASS
 RUN groupadd --gid 999 geoserver
-RUN useradd -ms /bin/bash --home /home/geoserver -p wtw.aC5paV.qU --uid 999 --gid 999 geoserver
+RUN useradd -ms /bin/bash --home /home/geoserver -p $(perl -e'print crypt("${USER_PASS:-geoserver}", "salt")') --uid 999 --gid 999 geoserver
```
Comment (Vampouille) [inline docker/ssh_data/Dockerfile]: Perl command is not working. Single quotes disable variable substitution so `${USER_PASS:-geoserver}` is not replaced. Instead, you can use echo to generate perl code and pipe result into perl :
```bash
RUN useradd -ms /bin/bash --home /home/geoserver -p $(echo "print crypt("${USER_PASS:-geoserver}", "salt")" | perl) --uid 999 --gid 999 geoserver
```

Comment (MaxiReglisse) [inline docker/ssh_data/Dockerfile]: you are right ! and you will set the new password when you will run the image : 
docker run -d --name georchestra_geodata_1 --env USER_PASS=another_password georchestra/ssh_data

**Comment by fvanderbiest** [comment]
@MaxiReglisse can you update the PR please ?

**Comment by fvanderbiest** [comment]
LGTM, leaving the merge to @Vampouille

**Pull request merged**